src/app/services/tests/AuthenticationService.spec.ts

```diff
@@ -440,7 +441,8 @@ describe('AuthenticationService', () => {
     });
     describe('launchAutomaticPresenceUpdate', () => {
         it('should be called and update user presence when user gets connected', async() => {
-            spyOn(RTDB, 'launchAutomaticPresenceUpdate').and.callThrough();
+            const connectivityDAO: ConnectivityDAO = TestBed.inject(ConnectivityDAO);
+            spyOn(connectivityDAO, 'launchAutomaticPresenceUpdate').and.callThrough();
```

Comment:
minute autisme: on décide de généraliser l'obligation de mettre .and.callThrough() par défaut explicitement, ou pas ?

Reply:
non ! sans callThrough ça n'appelle pas, donc c'est en fonction de ce que l'on souhaite !

Reply:
T'es sûr sûr sûr que sans .and.callThrough ça ne l'appelle pas ?
Si oui, ma question sur "devrions nous privilégier l'explicite à l'implicite" ?

Reply:
yep certain ! J'ai eu des comportements louches ("ptn pourquoi ça appelle pas, y'a pas de callFake ou autre !")
Sinon je dirais oui, privilégier .and.callThrough ou .and.returnValue etc.

Reply:
Ok, on tickette ça ? car t'as enlevé des callFake et returnValue dans un autre ticket à un endroit je crois.

Reply:
Ticketté ! Et yep, pour le handleError, vu qu'il doit retourner un `never`, ce cas là est un peu tricky mais ça sera à voir dans le ticket en question